### PR TITLE
Mod5 demo3

### PR DESCRIPTION
**Pull request by shalevz**
This demo requires just instructions.
Do I need to add **Verify that you have an azure account**? or should it be already in the setup guide?

### CODE REVIEW

**Comment by shalevz** [comment]
Added Avi as a reviewer

File: Instructions/20487D_MOD05_DEMO.md [hunk 1]
```diff
@@ -68,6 +68,74 @@
 
 
 
+# Lesson 2: Hosting Services in Azure Web Apps
+
+### Demonstration: Hosting ASP.NET Core Web APIs in Azure Web Apps
+
+1. Open a browser and browse to Azure portal at **portal.azure.com**.
```
Comment (AviAvni) [inline Instructions/20487D_MOD05_DEMO.md]: Open a browser and navigate to Azure portal

File: Instructions/20487D_MOD05_DEMO.md [hunk 2]
```diff
@@ -68,6 +68,74 @@
...
+2. If a page appears, prompting for your email address, type your email address, and then click Continue. Wait for the sign-in page to appear, enter your email address and password, and then click Sign In.
```
Comment (AviAvni) [inline Instructions/20487D_MOD05_DEMO.md]: If a page appears, asking for your email address

File: Instructions/20487D_MOD05_DEMO.md [hunk 3]
```diff
@@ -68,6 +68,74 @@
...
+
+    >**Note:** During the sign-in process, if a page appears prompting you to choose from a list of previously used accounts, select the account that you previously used, and then continue to provide your credentials.
```
Comment (AviAvni) [inline Instructions/20487D_MOD05_DEMO.md]: asking you

File: Instructions/20487D_MOD05_DEMO.md [hunk 4]
```diff
@@ -68,6 +68,74 @@
...
+
+3. If the **Windows Azure Tour** dialog box appears, click close (the **X** button).
+4. In the navigation blade, click **App Services**, click **Add**, click **Web App**, and then click **Create**.
```
Comment (AviAvni) [inline Instructions/20487D_MOD05_DEMO.md]: separate to different instructions and add some details

File: Instructions/20487D_MOD05_DEMO.md [hunk 5]
```diff
@@ -68,6 +68,74 @@
...
+6. Copy the **App name** value to any code editor.
+7. Click **App service plan/Location**, and then click **Create new**.
+8. From the **Location** drop-down list, select the region that is closest to your location.
+9. In the **App Service plan** input, enter **MyAppService**, and then click **OK**.
+10. Click **Create**. Wait for the web app to be created. Click the newly created web app.
+11. In the newly created web app blade, in the **Deployment** section, click **Deployment Credentials**.
+12. In the **FTP/deployment username** type a globally unique name.
+13. In the **Password** and **Confirm password** inputs, type a new password.
+
+    >**NOTE:** You will need the cardentials for the next steps. Copy them to any code editor.
```
Comment (AviAvni) [inline Instructions/20487D_MOD05_DEMO.md]: misspelling credentials

**Comment by shalevz** [comment]
done

**Pull request merged**